package.json

```
@@ -21,7 +21,7 @@
     "setup": "run-s db:generate db:push db:seed",
     "start": "cross-env NODE_ENV=production node ./build/server.js",
     "start:e2e": "cross-env NODE_ENV=production node --require dotenv/config ./build/server.js",
-    "test": "vitest",
+    "test": "NODE_OPTIONS=--require=dotenv/config vitest",
```

```suggestion
    "test": "cross-env NODE_OPTIONS=--require=dotenv/config vitest",
```
Without this, `npm test` will not run on windows